web/src/main/java/com/epam/esm/web/exception/ErrorHandler.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.esm.web.exception;
+
+public class ErrorHandler {

```

Comment:
According to name this class should contain logic and do something. But it's a simple DTO.

Reply:
wasn't thinking. thanks. renamed